event-controller-template.yml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Template
 labels:
-  template: event-watcher
+  template: event-controller
```

Comment:
do we want to spell it out to match the repo - i.e.; `openshift-event-controller`?